broker/config/crds/serviceflow.servicefabrik.io_v1alpha1_serialserviceflows.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: serialserviceflows.serviceflow.servicefabrik.io
+spec:
+  group: serviceflow.servicefabrik.io
+  names:
+    kind: Serialserviceflow
```

Comment:
Should it be in camel case ?

Reply:
Answered already. This how CRDs are supposed to be defined :-)

Reply:
**# kind is normally the CamelCased singular type. Your resource manifests use this**
you can see some examples [here](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/)

Reply:
For some reason it was earlier throwing exeptions during development. Now changed to camelcase and it seems to work. @amitmalav  do you remember this weird issue with camel casing i was facing during development?